radixdlt-regression/src/main/java/com/radixdlt/test/account/Account.java

```diff
@@ -175,6 +176,17 @@ public AID transfer(Account destination, Amount amount, Optional<String> message
         return TransactionUtils.nativeTokenTransfer(this, destination, amount, message);
     }
 
+    @Override
+    public AID transfer(Account destination, Amount amount, String rri, Optional<String> message) {
+        var request = message.map(s -> TransactionRequest.createBuilder(address)
+            .transfer(address, destination.getAddress(), amount.toSubunits(), rri)
+            .message(s)
+            .build()).orElseGet(() -> TransactionRequest.createBuilder(address)
+            .transfer(address, destination.getAddress(), amount.toSubunits(), rri)
+            .build());
+        return TransactionUtils.buildFinalizeAndSubmitTransaction(this, request, true);
+    }
```

Comment:
Seems something is wrong with formatting, all nested calls are aligned to same level.